srv/salt/ceph/monitoring/alertmanager/files/prometheus-webhook-snmp.conf.j2

```diff
@@ -0,0 +1,3 @@
+{%- for key, value in config.items() -%}
+{{ key }}: {{ value }}
```

Comment:
No need to iterate over a yaml dict to output a yaml dict.
```
{{config|yaml}}
```
will achieve the same.

Reply:
The output will be
```
{metrics: true, foo: bar}
```
but should be
```
metrics: true
foo: bar
```

Reply:
yeah fair...my mistake. _Should_ work though :wink: